Source/codec.cpp

```diff
@@ -115,9 +126,10 @@ void codec_encode(BYTE* pbSrcDst, DWORD size, int size_64, char *pszPassword)
 	}
 	memset(buf, 0, sizeof(buf));
 	SHA1Result(0, tmp);

```

Comment:
Thanks.  An egoistic question: if instead of `tmp` here you use `dst` which stays unused after the loop anyways:
```
	SHA1Result(0, tmp);
```
... does the bin-exactness of `codec_encode()` changes?  As count of local vars doesn't matter a lot, maybe the code could get rid of `tmp[]`.  Or does the name sound inapproriate for the logical context?

Reply:
Technically compiler could have optimized it, but in this case it's two distinct arrays allocated on stack so unfortunately it changes bin-exactness.